packages/app/src/cli/services/deploy/mode.ts

```diff
@@ -58,6 +58,9 @@ function displayDeployLegacyBanner(packageManager: PackageManager) {
           ],
         },
       },
+      '\n',
+      'All apps will be automatically upgraded on',
+      {bold: 'Sept 5, 2023'},
```

Comment:
In the screenshot in the issue, there's a period after the date. Also, it looks like this line is indented in your screenshot. Can the indent be removed?

Reply:
Oh nice catch! Missed the period.

I noticed the indentation too, although the screenshot if from this piece of code. Let me see if I can somehow get rid of it!

Reply:
I think putting the newline at the end of the last bullet point might achieve the right formatting.

Reply:
Right on!